inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -475,7 +474,7 @@ def match_references_hep(references):
 
 
 def match_references_based_on_flag(references):
```

Comment:
I think we're using this function only once, maybe we can use `match_references_hep` directly?